stable/ambassador/README.md

```diff
@@ -0,0 +1,118 @@
+# Ambassador
+
+Ambassador is an open source, Kubernetes-native [microservices API gateway](https://www.getambassador.io/about/microservices-api-gateways) built on the [Envoy Proxy](https://www.envoyproxy.io/).
+
+## TL;DR;
+
+```console
+$ helm install stable/ambassador
+```
+
+## Introduction
+
+This chart bootstraps an [Ambassador](https://www.getambassador.io) deployment on
+a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.7+
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm install --name my-release stable/ambassador
+```
+
+The command deploys Ambassador API gateway on the Kubernetes cluster in the default configuration.
+The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+$ helm delete --purge my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following tables lists the configurable parameters of the Ambassador chart and their default values.
+
+| Parameter                       | Description                                | Default                                                    |
+| ------------------------------- | ------------------------------------------ | ---------------------------------------------------------- |
+| `image.repository` | Image | `quay.io/datawire/ambassador`
+| `image.tag` | Image tag | `0.50.0`
+| `image.pullPolicy` | Image pull policy | `IfNotPresent`
+| `image.imagePullSecrets` | Image pull secrets | None
+| `daemonSet` | If `true `, Create a daemonSet. By default Deployment controller will be created | `false`
+| `env`  | Any additional environment variables for ambassador pods | `{}`
+| `replicaCount` | Number of Ambassador replicas  | `1`
+| `volumes` | Volumes for the ambassador service | None
+| `volumeMounts` | Volume mounts for the ambassador service | None
+| `resources` | CPU/memory resource requests/limits | None
```

Comment:
Most of the None defaults in README can be more specific to match the eact value in values.yaml (`{}`, `""`, `[]`). For example, this should be `{}`